### PR TITLE
Fuel System Improvements & New EICAS Gear Page

### PR DESCRIPTION
**Pull request by 747-4EVER**
Added the following fuel system features:

**Automatic Stab Fuel Transfer**
- Basic logic implemented for Auto Stab Fuel Transfer:
- Valves automatically sequence open when Centre Wing Tank fuel reaches 80,000LBS/11940GAL.
- This transfers fuel directly from STAB tank to CWT provided pumps are on.

**Automatic Reserve Tank 1&4 Transfer**
- The contents of Reserve Tanks 1&4 now automatically transfer to their respective Main Tanks when the Main Tank quantity 
   reaches 13,500LBS/2015GAL.

_Note: Although in the aircraft this transfer should be done by gravity, dummy pumps were added as a workaround as the gravity transfer function seems currently bugged._

**Added FUEL LOW CTR L/R EICAS message**
- Adds EICAS caution message FUEL LOW CTR that triggers when:
   Center wing tank is empty and;
   Respective CWT pump switches are still on
- This prompts manual reconfiguration by switching off CTR pumps

**Updated FUEL TANK/ENG EICAS message**
- Now also accounts properly for Reserve Tank quantity when triggering

**Added EICAS Gear page(Only semi functional):**
- Displays gear door status open/closed.
- Brake temp/tire pressure using dummy values until model can be developed.

### CODE REVIEW

**Comment by ninjomcs** [comment]
This makes my sim CTD when turning the APU on. Can anyone else confirm?

**Comment by 747-4EVER** [comment]
Sorry I'm away for a few days with no access to PC. Very strange though, didn't touch anything on the APU.

**Comment by 747-4EVER** [comment]
I think I have spotted the bug, the APUDCPump is accidentally defined twice (Line 272 and 277).
Must have missed it when merging into the 1.9.3 version. Unable to test at the moment though.

**Comment by 747-4EVER** [comment]
CTD bug caused by the APU has now been fixed with latest commit.

File: html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  					<rect rx="15" height="47" width="30" y="302" x="417" />
+  					<rect rx="15" height="47" width="30" y="250" x="463" />
+  					<rect rx="15" height="47" width="30" y="302" x="463" />
+						<line y2="273" x2="463" y1="273" x1="448" />
+  					<line y2="327" x2="463" y1="327" x1="448" />
+  					<ellipse ry="5" rx="5" cy="300" cx="455" />
+  					<line y2="294" x2="455" y1="273" x1="455" />
+  					<line y2="327" x2="455" y1="305" x1="455" />
+  					<rect rx="15" height="47" width="30" y="404" x="393" />
+ 					 	<rect rx="15" height="47" width="30" y="456" x="393" />
```
Comment (ninjomcs) [inline html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.html:31]: Please fix formatting (whole file)

File: html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        		
+            if ((GearDoorsOpen == 1) || (GearAnimClosed == 0)) {
+            	this.gearDoorOpenLines.style.visibility = "hidden";
+            	this.gearDoorClosedText.style.visibility = "visible";
+       	    } else {
+            	this.gearDoorOpenLines.style.visibility = "visible";   
+            	this.gearDoorClosedText.style.visibility = "hidden";
+            }
+      	}
+    }
```
Comment (ninjomcs) [inline html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.js:33]: Formatting

**Review by ninjomcs — CHANGES_REQUESTED** [review]

File: html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.html (hunk 1 above)
Comment (747-4EVER) [reply html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.html]: Formatting has been fixed (And apparently I need a new text editor...)

File: html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.js (hunk 2 above)
Comment (747-4EVER) [reply html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.js]: Fixed with latest commit

**Review by ninjomcs — APPROVED** [review]

**Pull request merged**